ios/FluentUI.Demo/FluentUI.Demo/Demos/TableViewCellFileAccessoryViewDemoController.swift

```diff
@@ -0,0 +1,425 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+import FluentUI
+
+// MARK: TableViewCellFileAccessoryViewDemoController
+
+class TableViewCellFileAccessoryViewDemoController: DemoController {
```

Comment:
It's more convenient to keep it a demo controller since there's logic in there that sets up a scroll view (scrollingContainer). We aren't using the container stack view but since this is test code, I'm not too concerned about optimizing the layout.